source/deployment/cluster.rst

```diff
@@ -265,6 +264,20 @@ In Mattermost v4.2 and later, a cluster leader election process assigns any sche
 
 The process is based on a widely used `bully leader election algorithm <https://en.wikipedia.org/wiki/Bully_algorithm>`_ where the process with the lowest node ID number from amongst the non-failed processes is selected as the "leader". 
 
+Job Server
+^^^^^^^^^^^^^^^^
+
+Mattermost schedules and completes periodic tasks via the `job server <https://docs.mattermost.com/administration/config-settings.html#jobs>`_. These tasks include:
+
+ - LDAP sync
+ - Data retention
+ - Compliance exports
+ - Elasticsearch indexing
+
+Run all job servers with ``--noschedule flag``, then set ``JobSettings.RunScheduled`` to ``true`` in config.json for all app servers in the cluster. The cluster leader at any given time will then be responsible for scheduling recurring jobs.
+
+For increased performance, use a standalone job server when running in High Availability mode.
```

Comment:
This is a bit confusing - it sounds like this is an alternative to what is described above, when really it is orthogonal to it.